OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -172,18 +181,14 @@ void Invalidate(IEnumerable<PPos> changed)
 					type = count > 0 ? ShroudCellType.Visible : ShroudCellType.Fog;
 				}
 
+				var oldResolvedType = resolvedType[uv];
 				resolvedType[uv] = type;
+				if (type != oldResolvedType)
+					OnShroudChanged((PPos)uv);
 			}
 
-			if (CellsChanged != null)
-				CellsChanged(changed);
-
 			var oldHash = Hash;
```

Comment:
As far as I can see this variable is no longer used?

Reply:
Removed.